srv/components/misc_pkgs/kafka/install.sls

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+{% import_yaml 'components/defaults.yaml' as defaults %}
+
+{% set kafka_version = pillar['commons']['version']['kafka'] %}
+
+Install Java:
+  pkg.installed:
+    - pkgs:
+      - java-1.8.0-openjdk
+      - java-1.8.0-openjdk-devel
```

Comment:
Not sure if we need devel. Usually it's avoided on Prod systems (comment the line).
Also, let's try to align this with S3Server which uses: `java-1.8.0-openjdk-headless`

Reply:
Addressed